eng/scripts/Create-APIView.ps1

```diff
@@ -310,6 +310,12 @@ function New-SwaggerAPIViewTokens {
     }
   }
 
+  if ($autoRestConfigInfo.Count -eq 0) {
+    LogWarning " No AutoRest configuration found for the changed swagger files in the current PR..."
+    Write-Host "##vso[task.complete result=SucceededWithIssues;]DONE"
```

Comment:
Why SucceedWithIssues? Should this just be Succeeded?

Reply:
Am using `SucceedWithIssues` so I can skip the next step at https://github.com/Azure/azure-rest-api-specs/blob/6c24168f4d2d737f070db7061731f7b6a8790c49/eng/pipelines/swagger-apiview.yml#L67 when the AutoRest Configuration is not valid to prevent the pipeline for failing. Otherwise PR author would have to update the readme to add the basic configuration section.

Mike is adding shared AutoRest parsing which I will switch to using when he is done https://github.com/Azure/azure-rest-api-specs/issues/33227